packages/opentelemetry-cloud-monitoring-exporter/src/external-types.ts

```diff
@@ -47,6 +47,12 @@ export interface ExporterOptions {
    * monitoring.googleapis.com:443.
    */
   apiEndpoint?: string;
+  /**
+   * Assume all metric descriptors have already been created and publish
+   * metrics without checking. This can prevent hitting a rate limit in Google
+   * when a large number of clients are all started up at the same time.
+   */
```

Comment:
Reworded comment but couldn't push to your branch. Please apply this if it looks OK to you

```suggestion
  /**
   * Disable calling CreateMetricDescriptor before sending time series to Cloud Monitoring.
   * Metric descriptors will be
   * {@link https://cloud.google.com/monitoring/custom-metrics/creating-metrics#auto-creation | auto-created}
   * if needed, but may be missing descriptions. This can prevent hitting a rate limit in
   * Cloud Monitoring when a large number of clients are all started up at the same time.
   */
```